### ISSUE TITLE
Sequences aren't being incremented

### ISSUE THREAD

**Issue by leosimoesp** [orientechnologies/orientdb#6969]
## OrientDB Version, operating system, or hardware.
- v2.0 SNAPSHOT[ ] - .18[ ] .17[ ] .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.1 SNAPSHOT[ ] - .16[ ] .15[ ] .14[ ] .13[ ] .12[ ] .11[ ] .10[ ] .9[ ] .8[ ] .7[ ] .6[ ] .5[ ] .4[ ] .3[ ] .2[ ] .1[ ] .0[ ]
- v2.2 SNAPSHOT[X ] - .rc1[ ] .beta2[ ] .beta1[ ]

## Operating System
- [X ] Linux
- [ ] MacOSX
- [ ] Windows
- [ ] Other Unix
- [ ] Other, name? 

## Expected behavior and actual behavior
When I executed java API call db.getMetadata().getSequenceLibrary().getSequence("mySeq").next() always returns the last value sequence and not new one.
## Steps to reproduce the problem
I've migrated from 2.2.10 version usin docker to 2.2.13

**Comment by leosimoesp** [comment]
![sequence-error-step1-2016-12-05 11-13-26](https://cloud.githubusercontent.com/assets/7965954/20886421/178e2adc-badd-11e6-8d21-28f0a788e20e.png)

![sequence-error-debug-step2-2016-12-05 11-16-02](https://cloud.githubusercontent.com/assets/7965954/20886472/3e42c700-badd-11e6-9c9f-20760c633a94.png)

![sequence-error-after-debug-step3-2016-12-05 11-18-41](https://cloud.githubusercontent.com/assets/7965954/20886499/657d19a6-badd-11e6-8593-f96e00dbff4e.png)

**Comment by lvca** [comment]
Are you using a remote connection?

**Comment by leosimoesp** [comment]
Hi Luca,

Yes, It's remote.



Em 27 de dez de 2016 12:19 PM, "Luca Garulli" <notifications@github.com>
escreveu:

**Comment by mmacfadden** [comment]
I think I might be seeing the same issue. I am doing:

```Java
OSequence seq = db.getMetadata().getSequenceLibrary().getSequence("seqname");
long next = seq.next();
```

Multiple calls to this are returning the same value, in this case 3, over and over. This is not happing in parallel threads so there is no concurrency issue. This is happening with a remote connection.

**Comment by mmacfadden** [comment]
@leosimoesp Did you ever figure anything out?  I tried calling `save()` on the sequence, but that did not work.  Also interestingly if I restart the database calling next gives me an incremented value the FIRST time I call it. Then from there always the same one. If I then restart again, it increments by one, then returns that again and gain.

**Comment by mmacfadden** [comment]
Pardon the scala code but here is some more info:

```scala
val seq = db.getMetadata().getSequenceLibrary().getSequence(AnonymousUsernameSeq)
println(seq.current())
val next = seq.next()
println(next)
println(seq.current())
```

Presently this prints:
9
10
10

If I re run the code a few seconds later I again get:

9
10
10

So it is odd that I get the current value, and it is 9, I get the next and the value now says 10. However if I come back through the code later (probably in a different thread or something) it is back to 9. I am using a connection pool if that makes a difference (the connection is being closed when I am done with it.).

**Comment by mmacfadden** [comment]
It seems like this might be related to the fix for #6842, specifically this commit:

https://github.com/orientechnologies/orientdb/commit/29505315c8e6ae0a83018ecb6560f7e946df6d55

**Comment by fixundfertig123** [comment]
@lvca Any news on this issues? Same happens to me on Windows and OrientDB 2.2.22. 
Unfortunately this prevents me of upgrading to 2.2.22 since the unqiue autoincrement is essential for my application.

**Comment by rajendrag** [comment]
This is reproducible on 2.2.23 as well. It is not working even with different transactions or same transaction.

**Comment by lvca** [comment]
We fixed a bug in sequences in 2.2.26. Please use latest hotfix available.
